### ISSUE TITLE
Restrict data extension property settings to specific properties 

### ISSUE THREAD

**Issue by fsteeg** [reconciliation-api/specs#14]
In the [lobid-gnd reconciliation service](https://lobid.org/gnd/reconcile) we have an `extends.property_settings` item to `select` if a property should be added as an ID or a label. This only makes sense for non-reconciled properties, since reconciled properties already have both ID and label. Since OpenRefine always shows the `configure` link, we explain in our documentation that this setting only makes sense for non-reconciled properties.

To allow clients to show configuration UI only for supported properties, perhaps it would make sense to add support for restricting settings to specific properties in [data extension property settings](https://reconciliation-api.github.io/specs/#dfn-data-extension-property-setting), e.g. with a `target` field. This could contain an array with the targeted property IDs, or some service-specific value (like `non_reconciled` in our example).

**Comment by wetneb** [comment]
Makes sense… but for services like Wikidata which have thousands of properties, it could be slightly impractical to list them all in the manifest, perhaps?

Overall I am not very happy with the way property settings work in the data extension protocol, it might be worth a more thorough overhaul in future versions, perhaps.
